content/en/docs/setup/additional-setup/customize-installation/index.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Customizing the installation configuration
+description: Describes how to customize installation configuration options.
+weight: 55
+aliases:
```

Comment:
I don't think we need an alias here as this is a new page (extracted from a page that still exists).